python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -141,18 +144,29 @@ def forward_cuda(
         key: torch.Tensor,
         offsets: Optional[torch.Tensor] = None,
     ) -> Tuple[torch.Tensor, torch.Tensor]:
-        from vllm import _custom_ops as ops
+        from sgl_kernel.ops import apply_rope_with_cos_sin_cache_inplace
```

Comment:
You should change the SGLang party in a follow-up PR after release a new version for sgl-kernel. Please don't combine them.

Reply:
Yes I am aware of that. This is just for my local testing